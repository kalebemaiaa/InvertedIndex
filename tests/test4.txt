QueryAnd(QueryWord("diplomata"), QueryOr(QueryWord("primeiro-ministro"), QueryWord("presidente")))
6403.raw
6334.raw
2851.raw
4446.raw
5400.raw
11823.raw
8767.raw
88.raw
1592.raw
626.raw
11882.raw
5782.raw
5862.raw
11995.raw
6384.raw
2895.raw
10853.raw
12010.raw
5304.raw
8758.raw
2301.raw
565.raw
8806.raw
2890.raw
58.raw
4811.raw
6256.raw
5293.raw
1678.raw
1816.raw
11858.raw
5687.raw
5042.raw
11472.raw
2675.raw
6315.raw
10957.raw
6215.raw
3326.raw
1487.raw
1314.raw
5905.raw
865.raw
325.raw
2430.raw
2101.raw
5934.raw
5749.raw
6309.raw
1524.raw
1490.raw
10842.raw
4738.raw
1268.raw
2793.raw
2272.raw
1905.raw
6258.raw
11182.raw
5581.raw
1909.raw
2441.raw
4600.raw
3613.raw
2801.raw
4883.raw
6105.raw
3814.raw
2294.raw
887.raw
10874.raw
11525.raw
6351.raw
1775.raw
12004.raw
4145.raw
1806.raw
10953.raw
11475.raw
4265.raw
3059.raw
2400.raw
5245.raw
568.raw
2164.raw
3072.raw
4338.raw
3629.raw
11528.raw
2695.raw
730.raw
4114.raw
1245.raw
1727.raw
10999.raw
5099.raw
3070.raw
101.raw
3693.raw
8766.raw
5830.raw
3830.raw
4937.raw
5178.raw
8877.raw
5345.raw
5308.raw
4817.raw
1297.raw
2538.raw
3035.raw
11187.raw
918.raw
4770.raw
8727.raw
2286.raw
4464.raw
2166.raw
4764.raw
6137.raw
11594.raw
5180.raw
734.raw
2066.raw
3597.raw
2131.raw
4790.raw
2866.raw
2257.raw
2821.raw
5012.raw
2031.raw
395.raw
11613.raw
1258.raw
2955.raw
4776.raw
8878.raw
265.raw
4867.raw
5390.raw
754.raw
11970.raw
4407.raw
4058.raw
2070.raw
2849.raw
3900.raw
6030.raw
1000.raw
2092.raw
1968.raw
1395.raw
3024.raw
5674.raw
4140.raw
4340.raw
47.raw
2227.raw
3285.raw
3112.raw
3115.raw
4854.raw
1421.raw
3627.raw
3345.raw
5737.raw
742.raw
999.raw
5754.raw
53.raw
11938.raw
473.raw
11298.raw
1943.raw
366.raw
4768.raw
4537.raw
4435.raw
2323.raw
11362.raw
5919.raw
940.raw
5413.raw
1579.raw
4833.raw
214.raw
1502.raw
5831.raw
1311.raw
1177.raw
1085.raw
11010.raw
1653.raw
336.raw
4375.raw
6150.raw
5544.raw
11207.raw
3213.raw
