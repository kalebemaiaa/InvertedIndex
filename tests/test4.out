727
1418
3588
11670
